test/enrichedFactories.js

```diff
@@ -169,6 +172,80 @@ export const makePickupSeries = data => {
   }
 }
 
+let votingIdCnt = 0
+export const makeVoting = data => {
+  return {
+    id: votingIdCnt++,
+    acceptedOption: 74,
+    expiresAt: addHours(new Date(), 106),
+    options: [{
+      affectedUser: 174,
+      id: 73,
+      sumScore: 7.0,
+      message: null,
```

Comment:
I removed the `message` field from the API just now, so that could be removed too